connectors/src/types/data_source_config.ts

```diff
@@ -0,0 +1,6 @@
+export interface DataSourceConfig {
+  systemAPIKey: string;
+  workspaceId: string;
+  dataSourceName: string;
```

Comment:
For now they are one and the same. That something we may want to change in the future but there is no "dataSourceId" just "dataSourceName"